plugins/chat/lib/chat_sdk/channel.rb

```diff
@@ -11,6 +11,9 @@ class Channel
     # @example Fetching messages from a channel with additional parameters
     #   ChatSDK::Channel.messages(channel_id: 1, guardian: Guardian.new)
     #
+    # @raise [RuntimeError] Raises an "Unexpected error" if the message retrieval fails for an unspecified reason.
+    # @raise [RuntimeError] Raises "Guardian can't view channel" if the user's permissions are insufficient to view the channel.
+    # @raise [RuntimeError] Raises "Target message doesn't exist" if the specified target message cannot be found in the channel.
```

Comment:
Not strictly related, but documented raised error for parity with new methods I added